lib/core/test/unit/Cardano/Wallet/Gen.hs

```diff
@@ -259,3 +272,48 @@ shrinkTxMetadataValue (TxMetaList xs) =
 shrinkTxMetadataValue (TxMetaNumber i) = TxMetaNumber <$> shrink i
 shrinkTxMetadataValue (TxMetaBytes b) = TxMetaBytes <$> shrinkByteString b
 shrinkTxMetadataValue (TxMetaText s) = TxMetaText <$> shrinkText s
+
+instance Arbitrary Natural where
```

Comment:
Best not to define this orphan instance and use the generator function directly.